packages/datetime/src/common/dateUtils.ts

```diff
@@ -110,6 +110,27 @@ export function isMonthInRange(date: Date, dateRange: DateRange) {
     return start <= day && day <= end;
 }
 
+export function isTimeInRange(time: Date, minTime: Date, maxTime: Date): boolean {
+    time = getDateOnlyWithTime(time);
+    minTime = getDateOnlyWithTime(minTime);
+    maxTime = getDateOnlyWithTime(maxTime);
+
+    const isTimeAfterMinTime = time.getTime() >= minTime.getTime();
+    const isTimeBeforeMaxTime = time.getTime() <= maxTime.getTime();
+
+    if (maxTime.getTime() <= minTime.getTime()) {
```

Comment:
hmm this sounds like an error state and should probably be validated by the component... 

likely safe to assume min <= max inside this function, though doesn't hurt to handle it like you're doing here.